### ISSUE TITLE
Use a 2D map for Sprite Animations

### ISSUE THREAD

**Issue by meronbrouwer** [han-yaeger/yaeger#62]
The current implementation uses a one dimensional map for Sprite Animation. This makes no sense and should be a 2D map.

This is also required for implementing the new animated logo from B302

Related to #51

**Comment by AbdelHajou** [comment]
Hey @meronbrouwer , are you still working on this? If not, I would like to pick this one up.

**Comment by meronbrouwer** [comment]
Ha @AbdelHajou , sadly I don't have the time right  now, so if you want to, you're more than welcome...

**Comment by AbdelHajou** [comment]
Do we really need a library for this? The library you mentioned in #51 is pretty small and it doesn't take much changes Yaeger's current implementation in order to support two-dimensional animations

**Comment by meronbrouwer** [comment]
You're probably right. It was someting I stumbled upon in the early days of Yaeger, so I just made the issue so I could have a look later on. I guess we won't need it, but it does have a nice feature (if I remember correctly) of supporting a Map, where you could name the different sprites.
I we could find a nice way of supporting this, it would be cool. Then again, anyone using a SpriteEntity could make such a map themselves.
So bottom line... feel free to do what you want. Try to go for a nice API, that still leaves enough freedom to the user.

**Comment by meronbrouwer** [comment]
btw @AbdelHajou do you have the correct privileges to drag the issue into the todo lane? If not, let me know.

**Comment by AbdelHajou** [comment]
@meronbrouwer I don't have permission to drag issues in workflow

**Comment by meronbrouwer** [comment]
@AbdelHajou I gave you write access, so it should be possible now.

**Comment by meronbrouwer** [comment]
I've moved this task to the Documentation lane while I'll implement it in the tutorial and the showcase
